src/Sarif/Autogenerated/AnnotatedCodeLocation.cs

```diff
@@ -39,16 +39,16 @@ public SarifNodeKind SarifNodeKind
         public string Message { get; set; }
 
         /// <summary>
-        /// Key/value pairs that provide additional details about the code location.
+        /// Key/value pairs that provide additional information about the code location.
```

Comment:
In various places we said "information about", "details about", and "information for". I made it "information about" everywhere.